app/sass/login-form.scss

```diff
@@ -0,0 +1,72 @@
+$base-color: #b73535;
+@mixin login-form-font($size) {
```

Comment:
Mixins should be moved to `tools` layer, eg. in `sass/tools/_mixins.scss`. Apart from that, this mixin isn't necessary, set a `font-family` on `body` and it will cascade nicely.